packages/wonder-blocks-announcer/src/__tests__/util/dom.test.ts

```diff
@@ -0,0 +1,133 @@
+import {screen, waitFor} from "@testing-library/react";
+import {
+    createRegionWrapper,
+    createDuplicateRegions,
+    createRegion,
+    removeMessage,
+} from "../../util/dom";
+
+jest.useFakeTimers();
+jest.spyOn(global, "setTimeout");
+
+describe("Announcer utility functions", () => {
+    describe("createRegionWrapper", () => {
+        test("it creates a polite region wrapper element", () => {
+            const element = createRegionWrapper("polite");
+
+            expect(element.tagName).toBe("DIV");
```

Comment:
**suggestion:** This assertion could be dropped as we can test that the element exists with the `id` we expect (`DIV` would be an implementation detail here that the test doesn't need to be aware of).
```suggestion
           
```